string str_box_ind
---
bool result
